### ISSUE TITLE
Scorecard not using settings from the scorecard.xml

### ISSUE THREAD

**Issue by edgarivanov** [siteadmin/Scorecard-Local-Instantiation#13]
I have been trying now for a while to configure Scorecard to use local installation of the C-CDA validator to no avail. I have deployed C-CDA validator locally and tested that it works fine via the web interface at http://localhost:8080/referenceccdaservice/swagger-ui.html
![image](https://user-images.githubusercontent.com/31533924/35629234-bf54410a-06a6-11e8-90be-41234d1d2811.png)

I have then configured Scorecard as per the Word document contained in this repo. Scorecard.xml looking like this (here I point 'igConformanceUrl' and certificationResultsUrl to the local installation of the C-CDA validator) 
![image](https://user-images.githubusercontent.com/31533924/35628490-bcb33f34-06a4-11e8-8003-61b12800bee2.png)

As soon as I submit document for the validation 
![image](https://user-images.githubusercontent.com/31533924/35629654-d536458a-06a7-11e8-9954-4735c784b5aa.png) 
I can see that scorecard is reaching out to the 50.112.65.223 on port 8443, which I assume is this service  https://ttpds.sitenv.org:8443/referenceccdaservice/swagger-ui.html 

![image](https://user-images.githubusercontent.com/31533924/35629169-969b23d2-06a6-11e8-9b86-e2d1765ee3e9.png)

On the image with Scorecard.xml file you can see all the options I have tried so far, regardless of what's in there Scorecard is still trying to talk to the remote service.

Is there a bug of some kind and specified configuration does not get applied from the scorecard.xml , or does Scorecard fall back to ttpds.sitenv.org when it can't make connection specified in the config file?

**Comment by drbgfc** [comment]
So sorry this was missed. I'm sure you've moved on or resolved this since - but if not - I would like to look into it or have someone else. It's a great question and a great writeup. If you setup the local ref val - which you did, I should think, like you stated, that you would be able to override the scorecard to use that version vs the remote version. However, it's possible, as you have noted, that this feature either doesn't exist or did not work properly. I did not do the scorecard to local version port so I am not sure. But I or someone else, will get to the bottom of it if you'd like. Again, understood if you have completely moved on by now!

**Comment by edgarivanov** [comment]
Hi, thanks for getting back. To be honest I don't know how much that is of an issue right now, obviously we have just moved on left it as it is. The original plan however was to put referenceccdavalidator and scorecard on a virtual machine in isolated environment where there would be no access to the internet. At that time I was just experimenting, in particular I was interested to see how connection presence to the internet affected document score after validation, sadly I can't now remember what was happening if I disconnected the internet connection. I can try and do some more tests with our current validator installation and see what happens if connection to the internet is banned, or if it not too much of work you can look in to it.

**Comment by drbgfc** [comment]
Right, aplogies again, as expected you have moved on. You certainly don't have to look into it for our sake - only if it benefits you. Otherwise, since no current user-interest, it would be of low-priority - but I would leave open so that we can look into it and if others have the issue they can reference this when there is a resolution. Going to assign this, thanks.
